core/ckeditor.js

```diff
@@ -33,6 +33,44 @@ CKEDITOR.instances = {};
  */
 CKEDITOR.document = new CKEDITOR.dom.document( document );
 
+/**
+ * Gets the element from DOM and checks if the editor can be instantiated on it.
+ * This function is available for internal use only.
+ *
+ * @private
+ * @since 4.12.0
+ * @returns {CKEDITOR.dom.element/null}
+ */
+CKEDITOR.getEditorElement = function( elementOrId ) {
```

Comment:
You mentioned that you had mixed feelings regarding that place and it's well justified - as it simply doesn't belong here.

We should be very careful when it comes down to adding members directly in the `CKEDITOR` namespace (even though you proposed adding it as a private member). The `CKEDITOR.editor` type seems to be _more_ suitable place for this logic.

Ideally we could put it in the constructor, right? because both creators use that, but that means we'd change a very popular API part - we shouldn't do that unless we really, really must.

With that in mind I think that easiest thing would be to put as a static method in `CKEDITOR.editor` type. Again it should have restricted visibility, as this is a function that is not really needed except for creators.